From e7496853e0f727cbaaec2195b6fd4507092de891 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Thu, 27 Aug 2015 09:35:02 +0900
Subject: [PATCH 72/85] arm64: configs: Disable CONFIG_ARCH_QCOM

This patch disable CONFIG_ARCH_QCOM because compile error occurs.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
---
 arch/arm64/configs/defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 5e0d495..814bb5b 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -42,7 +42,7 @@ CONFIG_ARCH_RCAR_GEN3=y
 CONFIG_ARCH_SEATTLE=y
 CONFIG_ARCH_TEGRA=y
 CONFIG_ARCH_TEGRA_132_SOC=y
-CONFIG_ARCH_QCOM=y
+# CONFIG_ARCH_QCOM is not set
 CONFIG_ARCH_SPRD=y
 CONFIG_ARCH_THUNDER=y
 CONFIG_ARCH_VEXPRESS=y
-- 
1.7.9.5

